### PR TITLE
ENH - Update theme and repo

### PR DESCRIPTION
**Pull request by trallard**
- :bento: Replace typefaces for Nebari brand ones
- :building_construction: Swap to pyproject.toml - use hatch
- :memo: Add contributing
- :wrench: Update login template
- :wrench: Update test to match our brand colours
- :truck: Rename QHub -> Nebari
- :bento: Replace images
- :lipstick: Update design assets
- :recycle: Refactor init file
- :lipstick: Change default styles
- :wrench: Replace setup.py for pyproject.toml

## What does this implement/fix?

_Put an `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [x] Breaking change (fix or feature that would cause existing features to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [x] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [ ] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

### Any other comments?

This is how this looks with the `test` page

JupyterHub

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,90 +1,198 @@
...
+- `navbar_text_color`: Navbar links and text color (`default: #1c1d26`)
+- `navbar_hover_color`: Hover color for navbar links (`default: #00a3b0`)
+
+> **Note**
+> The default colors, typefaces and logos are chosen based on the Nebari branding. You can find more about this on the [Nebari design repository](https://github.com/nebari-dev/nebari-design).
+
+:computer: You can see an example of these variables and the configuration in [`test_jupyterhub_config.py`](test_jupyterhub_config.py).
+
+Nebari internal options:
+- `cdsdashboards_enabled`
```
Comment (viniciusdc) [inline README.md]: using the `cdsdashboards_enabled` option will require the jupyterhub environment to contain `cdsdashboards` to work correctly. Maybe mentioning the package or a link here.. what do you think?

File: README.md [hunk 2]
```diff
@@ -1,90 +1,198 @@
...
+- `hub_title`: The title of the JupyterHub instance. This will be displayed in the header of the login page. (H1 heading, `default: Welcome to Nebari`)
+- `hub_subtitle`: Subtitle shown under the main instance Title. (H2 heading, `default: ""`)
+- `welcome`: Welcome message on the login page (`default:  Welcome to Nebari. For more information about Nebari, visit <a href="https://nebari.dev/">https://nebari.dev</a>`)
+- `logo`: Logo displayed on the navbar (`default: Nebari logomark`)
+- `primary_color`: Primary color (`default: #cb39ed`)
+- `secondary_color`: Secondary color (`default: #2bd1c5`)
+- `accent_color`: Accent color (`default: #eda61d`)
+- `text_color`: Text color (`default: #1c1d26`)
+- `h1_color`: H1 color, main Title in login page (`default: #0f1015`)
+- `h2_color`: H2 color, subtitle in login page (`default: #0f1015`)
```
Comment (viniciusdc) [inline README.md:158]: There is also the `display_version` configuration

**Review by viniciusdc — COMMENTED** [review]
Looks great! I left some comments and questions :smile:

File: README.md (hunk 1 above)
Comment (trallard) [reply README.md]: 👍🏽  - I was unsure as this is entirely "nebari specific" but you are correct we should add this

**Pull request merged**